frontend_tests/node_tests/narrow_activate.js

```diff
@@ -82,6 +82,7 @@ function test_helper() {
     stub('ui_util', 'change_tab_to');
     stub('unread_ops', 'process_visible');
     stub('compose', 'update_closed_compose_buttons_for_stream');
+    stub('compose', 'update_closed_compose_buttons_for_other_narrows');
```

Comment:
maybe lets call this update_closed_compose_buttons_for_non_stream_narrows?

Reply:
Should it be `update_compose_buttons_for_stream_narrows` or `update_closed_compose_buttons_for_stream_narrows`?

Reply:
not sure; we can wait for Tim for that.

Reply:
So, the addition of the popover meant that these functions weren't needed anymore. I've removed them entirely.